src/main/java/org/jabref/gui/importer/FileFilterConverter.java

```diff
@@ -0,0 +1,41 @@
+package org.jabref.gui.importer;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.SortedSet;
+import java.util.stream.Collectors;
+
+import javafx.stage.FileChooser;
+
+import org.jabref.logic.exporter.Exporter;
+import org.jabref.logic.importer.Importer;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.logic.util.FileType;
+
+public class FileFilterConverter {
+
+    private FileFilterConverter() {
+    }
+
+    private static FileChooser.ExtensionFilter convertImporter(String description, Collection<Importer> formats) {
+        List<FileType> fileTypes = formats.stream().map(Importer::getFileType).collect(Collectors.toList());
+        List<String> flatExtensions = fileTypes.stream().flatMap(extList -> extList.getExtensions().stream())
+                .map(ending -> "*." + ending)
+                .collect(Collectors.toList());
+
+        return new FileChooser.ExtensionFilter(description, flatExtensions.toArray(new String[flatExtensions.size()]));
+    }
+
+    public static Optional<Importer> getImporter(FileChooser.ExtensionFilter extensionFilter, Collection<Importer> importers) {
+        return importers.stream().filter(importer -> importer.getDescription().equals(extensionFilter.getDescription())).findFirst();
+    }
+
+    public static Optional<Exporter> getExporter(FileChooser.ExtensionFilter extensionFilter, Collection<Exporter> exporters) {
```

Comment:
Either you move the whole class to a new "meta" package, or you move the getExporter method to another class
Atm it is still in import and I would not expect an export class here

Reply:
Moved to `gui.util`